### PR TITLE
Fix `release-download-count` feature

### LINKED ISSUE #5147: `release-download-count` no longer displays on the Releases overview tab
### Please ensure:

- [X] The bug is caused by Refined GitHub. It doesn't happen if I disable the extension.

### Include in this issue:

- [X] Screenshots/video/gif demonstrating the bug, if it’s visual

### Description

Currently, when visiting any repo's `/Releases` page, the assets download count will not show.  
It still shows when you go to a specific tag and view one (pre-)release by itself.

Considered this as a bug as it used to display on the `/Releases` page a few weeks back.

## Expected Behaviour
Display the assets's download count next to it even when scrolling through the `/Releases` page

### Not working on the general `/Releases` page
Download Count Lack Image

### Working when visiting a specific tag
Download Count On Tag

### Example URL

https://github.com/AudioBand/AudioBand/releases

### Browser(s) used

Firefox, Google Chrome

### PR DESCRIPTION
**Pull request by cheap-glitch**
Fixes #5147 

## Test URLs

 * [Release list](https://github.com/AudioBand/AudioBand/releases)
 * [Single release](https://github.com/AudioBand/AudioBand/releases/tag/v1.0.3)
 * [Release list (with several artifact per release)](https://github.com/fish-shell/fish-shell/releases)

## Screenshot

image

### CODE REVIEW

File: source/features/release-download-count.tsx [hunk 1]
```diff
@@ -51,9 +51,9 @@ async function init(): Promise<void | false> {
 		const name = select('.Box svg.octicon-tag ~ span')!.textContent!.trim();
 		releases.set(name, select('.Box-footer')!);
 	} else {
-		for (const release of select.all('.js-release-expandable')) {
+		for (const release of select.all('[data-test-selector="release-card"] > .Box')) {
 			if (!select.exists('.octicon-package', release)) {
-				return;
+				continue;
```
Comment (cheap-glitch) [inline source/features/release-download-count.tsx:56]: Mea culpa :grimacing:

File: source/features/release-download-count.tsx [hunk 2]
```diff
@@ -51,9 +51,9 @@ async function init(): Promise<void | false> {
 		const name = select('.Box svg.octicon-tag ~ span')!.textContent!.trim();
 		releases.set(name, select('.Box-footer')!);
 	} else {
-		for (const release of select.all('.js-release-expandable')) {
+		for (const release of select.all('[data-test-selector="release-card"] > .Box')) {
```
Comment (cheap-glitch) [inline source/features/release-download-count.tsx:54]: This is the least worst selector I could find as a replacement.

File: source/features/release-download-count.tsx (hunk 1 above)
Comment (fregante) [reply source/features/release-download-count.tsx:56]: Pretty standard mistake 😅

**Review by fregante — APPROVED** [review]

**Comment by svr333** [comment]
Thanks :)

**Comment by balgable** [comment]
Hi

**Pull request merged**